File: app/src/main/java/com/an/moviesample/AppUtils.java
package com.an.moviesample;

import android.content.Context;
import android.graphics.Point;
import android.os.Build;
import android.view.Display;
import android.view.WindowManager;

public class AppUtils {

    public static int getScreenWidth(Context mContext) {
        boolean width = false;
        WindowManager wm = (WindowManager)mContext.getSystemService("window");
        Display display = wm.getDefaultDisplay();
        int width1;
        if(Build.VERSION.SDK_INT > 12) {
            Point size = new Point();
            display.getSize(size);
            width1 = size.x;
        } else {
            width1 = display.getWidth();
        }

        return width1;
    }

    public static int getScreenHeight(Context mContext) {
        boolean height = false;
        WindowManager wm = (WindowManager)mContext.getSystemService("window");
        Display display = wm.getDefaultDisplay();
        int height1;
        if(Build.VERSION.SDK_INT > 12) {
            Point size = new Point();
            display.getSize(size);
            height1 = size.y;
        } else {
            height1 = display.getHeight();
        }

        return height1;
    }
}

File: app/src/main/java/com/an/moviesample/AppUtils.kt
package com.an.moviesample

import android.content.Context
import android.graphics.Point
import android.os.Build
import android.view.Display
import android.view.WindowManager

object AppUtils {

    fun getScreenWidth(mContext: Context): Int {
        val wm = mContext.getSystemService(Context.WINDOW_SERVICE) as WindowManager
        val display = wm.defaultDisplay
        val width1: Int
        width1 = if (Build.VERSION.SDK_INT > 12) {
            val size = Point()
            display.getSize(size)
            size.x
        } else {
            display.width
        }

        return width1
    }

    fun getScreenHeight(mContext: Context): Int {
        val wm = mContext.getSystemService(Context.WINDOW_SERVICE) as WindowManager
        val display = wm.defaultDisplay
        val height1: Int
        height1 = if (Build.VERSION.SDK_INT > 12) {
            val size = Point()
            display.getSize(size)
            size.y
        } else {
            display.height
        }

        return height1
    }
}
